### PR TITLE
Explicitly add note to the docs that admin user needs to be created b…

### LINKED ISSUE #19145: Cannot produce an access token for the admin console
### Describe the bug

Following the instructions outlined here:
https://github.com/keycloak/keycloak-documentation/blob/main/server_development/topics/admin-rest-api.adoc

Always 401. Both steps

keycloak-17.0.1

### Version

keycloak-17.0.1

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
…efore using admin REST API

closes #19145

### CODE REVIEW

**Comment by mposolda** [comment]
@andymunro Could you please review this small PR? Thanks!

File: docs/documentation/server_development/topics/admin-rest-api.adoc [hunk 1]
```diff
@@ -10,6 +10,8 @@ You can obtain a token by enabling authentication for your application using {pr
 
 ==== Authenticating with a username and password
 
+NOTE: Following example assume that you created user `admin` with password `password` in the `master` realm as shown in the link:{gettingstarted_link}[{gettingstarted_name}] tutorial.
```
Comment (andymunro) [inline docs/documentation/server_development/topics/admin-rest-api.adoc:13]:
```suggestion
NOTE: The following example assumes that you created the user `admin` with the password `password` in the `master` realm as shown in the link:{gettingstarted_link}[{gettingstarted_name}] tutorial.
```

**Review by andymunro — COMMENTED** [review]
Just one change @mposolda

**Comment by mposolda** [comment]
@andymunro Applied, Thanks!

**Review by andymunro — APPROVED** [review]

**Pull request merged**